src/Styleguide/Utils/HorizontalPadding.tsx

```diff
@@ -0,0 +1,26 @@
+import { space, SpacingUnit } from "@artsy/palette"
+import styled from "styled-components"
+import { media } from "../../Components/Helpers"
+
+interface PaddingProps {
+  sizeXS?: SpacingUnit | 0
+  size?: SpacingUnit | 0
+}
+
+const responsivePadding = media.sm<PaddingProps>`
+  padding-right: ${p => space(p.sizeXS as SpacingUnit) || 0}px;
+  padding-left: ${p => space(p.sizeXS as SpacingUnit) || 0}px;
+`
+
+export const HorizontalPadding = styled.div.attrs<PaddingProps>({})`
+  padding-right: ${p => space(p.size as SpacingUnit) || 0}px;
+  padding-left: ${p => space(p.size as SpacingUnit) || 0}px;
```

Comment:
Is `p.size` being cast here due to the possibility that it includes `0`? In the interface, could remove `| 0` and simplify fall back to `p.size && space(p.size) || 0`. 

Another approach would be 

```js
export const HorizontalPadding = styled.div<PaddingProps>`
  ${p => p.size && css`
      padding-right: ... 
      padding-left: ... 
   `
  }
`
```